README.md

```diff
@@ -0,0 +1,5 @@
+1. sing up or sing in bitbucket
```

Comment:
Better to use ordered lists, see markdown basic syntax for reference https://guides.github.com/features/mastering-markdown/